staging/src/k8s.io/apiserver/pkg/apis/example2/install/BUILD

```diff
@@ -21,7 +21,7 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    srcs = ["roundtrip_test.go"],
+    srcs = ["install_test.go"],
```

Comment:
Looks like  we can update the same here https://github.com/kubernetes/kubernetes/blob/2cbb16bc8dd456c5db72c1667926abdbc87c32c7/staging/src/k8s.io/apiserver/pkg/apis/example/install/BUILD#L24

Reply:
Oh, I missed that.
Thanks for your review!